src/examples/counter-provider/CounterProvider.js

```diff
@@ -1,10 +1,15 @@
 import React from "react";
-import { useCountGovernor } from "../counter/Counter";
+import { useGovernor } from "../../";
+import { counter_actions } from "../counter/CounterActions";
 
 const CounterContext = React.createContext();
 
+const initialState = {
+  count: 0
+};
+
```

Comment:
Should we promote the example where `initialState` is coming from `"../counter/CounterActions"`?

`import { counterActions, initialState } from "../counter/CounterActions";`

Also small nit using `counterActions` vs `counter_actions` to keep casing consistent

Reply:
example here where i'm just exporting a `counterState` and `counterActions`.. interestingly, when I remove both values from `useGovernor()` call.. it's not throwing an error https://codesandbox.io/s/1y494lmqq3

Reply:
It should not throw an error when you call it with no args; I have default params as empty objects (no initial state and no actions, which is fine but useless).

Reply:
It should not throw an error when you call it with no args; I have default params as empty objects (no initial state and no actions, which is fine but useless).